#!/usr/bin/env python
PACKAGE = "orp"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

#gen.add(name, type, 0, description, default, min, max
# EXAMPLE
#gen.add("variable",                  int_t,    0, "Edge detection threshold", 50,1,100)

##############################################################################


gen.add("max_iterations", double_t, 0, "", 10000, 1, 1000000)
gen.add("distance_threshold", double_t, 0, "", 0.02,0,0.5)

##############################################################################
exit(gen.generate(PACKAGE, "orp", "CornerClassifier"))
